### DISCUSSION TITLE
Nextcloud-AIO through nat64 gateway

### DISCUSSION THREAD

**Discussion by melbruki** [nextcloud/all-in-one#3976] [Questions] [answered]
Hello will nextcloud-aio work through a nat64 gateway . I have a IPv6 only k8s cluster and have a nat64x46 gateway that provides access to my IPv4 clients. Will my IPv4 clients using nextcloud and nextcloud talk have any issues ?

**Comment by szaimen** [comment, marked as answer]
Hi, you will need to test this yourself.
